scripts/nginx/sonarrv3.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Nginx conf for Sonarr v3
+# Flying sausages 2020
+master=$(cut -d: -f1 < /root/.master.info)
+
+cat > /etc/nginx/apps/sonarrv3.conf <<SONARR
+location /sonarr {
+  proxy_pass        http://127.0.0.1:8989/sonarr;
+  proxy_set_header Host \$proxy_host;
+  proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+  proxy_set_header X-Forwarded-Proto \$scheme;
+  proxy_redirect off;
+  auth_basic "What's the password?";
+  auth_basic_user_file /etc/htpasswd.d/htpasswd.${master};
+}
+SONARR
+
+isactive=$(systemctl is-active sonarr)
+
+if [[ $isactive == "active" ]]; then
+  systemctl stop sonarr
+fi
+
+cat > /var/lib/sonarr/config.xml <<SONN
```

Comment:
I do not think we should be using `/var/lib/sonarr` for sonarr storage. I know this is default but nzbdrone storage has always been held in .config. `~/.config` already contains the data of the majority of applications and I don't really want to move apps that are already there out of there.

Reply:
This should do it, leaving this here for later https://github.com/Sonarr/Sonarr/blob/phantom-develop/distribution/debian/config#L11

But I'd say I'd rather leave this as unmodified as possible given it's in the package manager. It's all documented too. Let me know if you won't budge on this and I'll just change it though.

Reply:
Perhaps we should open it up to community review as I do not think that changing an established config location is a great idea. People are already used to configs being stored here, I just don't see the reason to change this.

Reply:
I would say it's not getting changed as it's a new package completely. We could maybe symlink it in there for convenience's sake..?

Reply:
Changed to ~/.config